plugins/inventory/now.py

```diff
@@ -0,0 +1,443 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import os
+
+from ansible.errors import AnsibleParserError
+from ansible.plugins.inventory import BaseInventoryPlugin, to_safe_group_name
+
+from ..module_utils.client import Client
+from ..module_utils.table import TableClient
+
+
+DOCUMENTATION = r"""
+    name: servicenow.itsm.now
+    plugin_type: inventory
+    author:
+      - Manca Bizjak (@mancabizjak)
+      - Miha Dolinar (@mdolin)
+      - Tadej Borovsak (@tadeboro)
+    short_description: Inventory source for ServiceNow table records.
+    version_added: "2.10"
+    description:
+        - Builds inventory from ServiceNow table records.
+        - Requires a configuration file ending in C(now.yml) or C(now.yaml).
+        - The plugin sets host variables denoted by I(columns).
+    options:
+        plugin:
+            description:
+                - The name of the ServiceNow Inventory Plugin.
+                - This should always be C(servicenow.itsm.now).
```

Comment:
We can make this a constant variable. Since we are not allowing any other value.

Reply:
There is `choices: [ servicenow.itsm.now ]` on L39 that limits the set of allowable values. And the `plugin` option must be there so that the `auto` plugin knows to load our inventory plugin, right? Is there another way to do this, what exactly did you have in mind?